web/src/main/java/org/fao/geonet/Geonetwork.java

```diff
@@ -419,6 +420,8 @@ private void importDatabaseData(final ServiceContext context) {
                 Log.error(Geonet.DB, "Error occurred while trying to execute SQL", t);
                 throw new RuntimeException(t);
             }
+
+            context.getBean(IsoLanguagesMapper.class).init();
```

Comment:
maybe is renit better, as init would append entries at the end of existing  ones, the are maybe figure where entries are already there whene this init call is done.